### PR TITLE
Pm google auth

### PR DESCRIPTION
**Pull request by petermoyano**

### CODE REVIEW

File: src/lib/auth.ts [hunk 1]
```diff
@@ -26,36 +28,62 @@ export const config = {
           typeof email !== "string" ||
           email.length < 3 ||
           typeof password !== "string" ||
-          email.length < 3
+          password.length < 3
```
Comment (andvirga) [inline src/lib/auth.ts]: We should enforce using password length with at least 8 characters.
```suggestion
          password.length < 8
```

**Review by andvirga — APPROVED** [review]
LGTM, only left one comment about password length

**Review by andvirga — APPROVED** [review]
LGTM

**Comment by brunomembrado** [comment]
I'm trying to do the normal login with admin credentials in this pr preview and I'm not being able. I saw that the password length was extended. Therefore, I don't know if that has something to do with it.
Production: (Not problem with loggin in)
Screenshot 2024-10-07 at 11 59 42 AM
This PR Preview:
Screenshot 2024-10-07 at 12 03 57 PM

**Pull request merged**